small placeholder file and link to git repo api docs (#882)
elastic/observability-docs: docs/en/ingest-management/fleet/fleet-api-docs.asciidoc

[[fleet-api-docs]]

> = {{fleet} API Docs}

Reviewer: ```suggestion
= {fleet} APIs
```

Author: : / lol, I wasn't sure how this was expecting to resolve.  i couldn't have made this pr any more complicated (and wrong) for how small it is.

> {fleet} is fully supported by an API layer.  Anything you can do in the UI, can be
> done using the API.  Indeed, the UI consumes the API and proves out its usage!

Reviewer: ```suggestion
{fleet} is fully supported by an API layer. Any actions you can perform
through the {fleet} UI are also available through the API. In fact, the UI
consumes the API.
```

Author: if you don't prefer the language I used, it was copied from another page I think, I can look for it and update it there for us too.  Thanks for the rewrite.

Reviewer: @EricDavisX I think the language has a few problems. It's too vague because it doesn't qualify which UI we're talking about. Generally it's good to avoid words like "thing" because they are too broad and don't translate well. Also, the word "anything" tends to over-promise. The last line seems too enthusiastic for docs, but would probably be OK in the blog post.

> Please see the {kib} repo
> https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/common/openapi/README.md[{fleet} API Documentation file]
> for more details.

Reviewer: ```suggestion
Please refer to the
https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/common/openapi/README.md[{fleet} OpenAPI file] for more details.
```

> //needed to render tab widgets
> include::{tab-widgets}/code.asciidoc[]

Reviewer: ```suggestion
```

Reviewer: page is not required because it doesn't use the tabbed widgets

[[fleet-api-docs]]
= {fleet} APIs

{fleet} is fully supported by an API layer. Any actions you can perform

> through the {fleet} UI are also available through the API. In fact, the UI
> consumes the API.

Reviewer 2: ```suggestion
through the {fleet} UI are also available through the API.
```

suggest to remove this last bit as well, that's how it generally works :)

Author: indeed.  :)

through the {fleet} UI are also available through the API. In fact, the UI
consumes the API.

Please refer to the

> https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/common/openapi/README.md[{fleet} OpenAPI file] in the {kib} repo for more details.

Reviewer: ```suggestion
https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/fleet/common/openapi/README.md[{fleet} OpenAPI file] in the {kib} repo for more details.
```

Reviewer: Sorry there is one last thing I forgot to fix. We need to use a variable here, or the URL will always point to master and not necessarily reflect the correct version.
